Libraries/src/Amazon.Lambda.DynamoDBEvents/DynamoDBEvent.cs

```diff
@@ -21,15 +18,315 @@ public class DynamoDBEvent
+            /// An attribute of type Boolean. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"BOOL": true</c>
+            /// </para>
+            /// </summary>
+            public bool BOOL { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type Binary Set. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"BS": ["U3Vubnk=", "UmFpbnk=", "U25vd3k="]</c>
+            /// </para>
+            /// </summary>
+            public List<MemoryStream> BS { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type List. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"L": [ {"S": "Cookies"} , {"S": "Coffee"}, {"N": "3.14159"}]</c>
+            /// </para>
+            /// </summary>
+            public List<AttributeValue> L { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type Map. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"M": {"Name": {"S": "Joe"}, "Age": {"N": "35"}}</c> 
+            /// </para>
+            /// </summary>
+            public Dictionary<string, AttributeValue> M { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type Number. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"N": "123.45"</c>
+            /// </para>
+            ///  
+            /// <para>
+            /// Numbers are sent across the network to DynamoDB as strings, to maximize compatibility
+            /// across languages and libraries. However, DynamoDB treats them as number type attributes
+            /// for mathematical operations.
+            /// </para>
+            /// </summary>
+            public string N { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type Number Set. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"NS": ["42.2", "-19", "7.5", "3.14"]</c>
+            /// </para>
+            ///  
+            /// <para>
+            /// Numbers are sent across the network to DynamoDB as strings, to maximize compatibility
+            /// across languages and libraries. However, DynamoDB treats them as number type attributes
+            /// for mathematical operations.
+            /// </para>
+            /// </summary>
+            public List<string> NS { get; set; }
+
+            /// <summary>
+            /// <para>
+            /// An attribute of type Null. For example:
+            /// </para>
+            ///  
+            /// <para>
+            ///  <c>"NULL": true</c> 
+            /// </para>
+            /// </summary>
+            public bool NULL { get; set; }

```

Comment:
Use the nullable type. Here use `bool?`.

Reply:
Updated in https://github.com/aws/aws-lambda-dotnet/commit/3bd10bc4900e4d21ebba4d47fe9700cd0076a74a#diff-03c240223072abf4e9dea6645203768f84845f84f54ea472b360aeff7d3960cfR307